src/formatter/ExpressionFormatter.ts

```diff
@@ -354,20 +389,8 @@ export default class ExpressionFormatter {
       case TokenType.OR:
       case TokenType.XOR:
         return this.formatLogicalOperator(node);
-      case TokenType.RESERVED_KEYWORD:
-      case TokenType.RESERVED_FUNCTION_NAME:
-      case TokenType.RESERVED_PHRASE:
-      case TokenType.THEN:
-        return this.formatKeyword(node);
-      case TokenType.CASE:
-        return this.formatCaseStart(node);
-      case TokenType.END:
-        return this.formatCaseEnd(node);
-      case TokenType.WHEN:
-      case TokenType.ELSE:
-        return this.formatCaseWhenOrElse(node);
       default:
-        throw new Error(`Unexpected token type: ${node.tokenType}`);
+        return this.formatKeyword(node);
```

Comment:
Removed the error to turn this method into a more generic one that's used for formatting all kinds of keywords.